### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by moiseevaleksey**

### CODE REVIEW

**Comment by moiseevaleksey** [comment]
@MoiseevaTat 
1) Rename js file proper way 
2) Remove unused code from js
3) What is the difference between decision and solution?

**Comment by MoiseevaTat** [comment]
A decision is a choice you can make,  a solution would be a way of fixing a problem.

File: task1.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ 	z = -- x - y * 5 ; 
+ 	console.log('z=' + z);
+ 	y /= x + 5 % z ;
+ 	console.log('y=' + y);  
+ 	z = x++ + y * 5 ; 
+ 	console.log('z=' + z);
+ 	x = y - x++ * z ; 
+ 	console.log('x=' + x);
+}
+foo (x, y, z);
```
Comment (moiseevaleksey) [inline task1.js]: 1) Make sure that you are using correct code styling, I mean that you use wrong paddings.
2) Calculate these formulas on the paper or somewhere else and compare the results.

File: task1.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ 
+ 
+var x=6, y=15, z=4;
+console.log((x+y+z)/3);
```
Comment (moiseevaleksey) [inline task1.js]: Create separate function for this calculation

**Comment by MoiseevaTat** [comment]
I checked the correctness of the calculations with the breakpoint

**Pull request merged**